.github/workflows/build-publish.yaml

```diff
@@ -80,6 +87,9 @@ jobs:
           context: ./
           push: true
           file: ${{ matrix.package.path }}/Dockerfile.prod
+          build-args: |
+            ${{ matrix.package.identifier == 'boutique:frontend' && format('VITE_API_BASE_URL={0}', matrix.package.api_base_url) }}
+            ${{ matrix.package.identifier == 'boutique:frontend' && format('VITE_CURRENCY={0}', matrix.package.currency) }}
```

Comment:
This results in:
```
/usr/bin/docker buildx build --build-arg false --build-arg false 
```

Note the `--build-arg false`. Add `format('VITE_API_BASE_URL={0}', matrix.package.api_base_url) || ''` again. This way we are not going to pass `false` multiple times as a build argument.

Workflow run when using ` || ''`:  https://github.com/interledger/testnet/actions/runs/11217123924/job/31178030911
Workflow run when not using ` || ''`: https://github.com/interledger/testnet/actions/runs/11218519020/job/31182550051

Reply:
good catch, thanks!